data/projects/bear_updates.md

```diff
@@ -1,26 +1,42 @@
 This project is about improving the general quality of coala's linter bears.
 
-Linter bears in coala are oftentimes not receiving enough love.
-They work but not necessarily on all platforms. They are documented, but not
-necessarily understandable. They are tested but lack corner cases or
+Since it is very easy to write them even for very inexperienced contributors, linter bears in coala often suffer from mediocre quality,
+especially once they made it into the codebase.
+They do work, but not necessarily on all platforms or for all usecases. They are documented, but not
+necessarily understandable. They are tested, but lack corner cases or
 exploratory tests. [See existing issues](https://github.com/coala/coala-bears/issues?q=is%3Aopen+is%3Aissue+label%3Aarea%2Flintbears)
 
-Task of this project is to solve those issues for as many bears as possible and
-derive strategies to avoid those issues in the future.
+Students working on this project familiarize themselves in-depth with the working of the `linter`
+decorator and the existing linter bears. They identify systematic issues with the way linter bears are
+developed and maintained and work to fix those issues and improve and amend [existing documentation](http://api.coala.io/en/latest/Developers/Writing_Linter_Bears.html)
+to avoid them in the future.
+
+Possible areas to investigate include:
+
+* Tests: As mentioned above, tests for linter bears are often kept to a minimum. They use hardcoded files embedded into the test file, which makes it cumbersome and ugly to add more test cases.
+* Support for underlying tool: The support for the underlying tool is mostly left to the contributers discretion. Students could come up with a guideline on how well a tool's options must be supported and documented or even automated ways to test for this (maybe via a bear?).
 
 #### Milestones
 
-##### GSoC 2017 Community Bonding
+##### PREPARATION/BONDING
+
+ * The student is familiar with the `linter` decorator and the ecosystem of available linter bears.
+ * Systematic issues with the way bears are developed and maintained are identified.
+ * For each of the identified issues a set of goals is defined which should be reached by the end of the program.
+
+##### CODING PHASE 1
 
- * For all issues, discusions about how to solve them properly and sustainably
- have to be resolved.
- * Core changes and guidelines for enforcing higher bear quality are documented
- in [the coala repository](https://api.coala.io).
+ * Ideas for how to solve each of the indetified issues are drafted.
+ * Changes to the [linter bear documentation](http://api.coala.io/en/latest/Developers/Writing_Linter_Bears.html) to improve quality of future bears are proposed.
+ * A list of bears which suffer from any of the identified issues is composed.
```

Comment:
Do you have concrete suggestions? I don't want to write "some work should be merged". Also update of the docs is merged work.